BUILDING.md

```diff
@@ -210,8 +247,10 @@ This means you will need to have external copies of these libraries available
 on your system. This [example](test/distrib/cpp/run_distrib_test_cmake.sh) shows
 how to install dependencies with cmake before proceeding to installing gRPC itself. 
 
+> **Note**
+> All of gRPC's dependencies need to be already installed. 
```

Comment:
This is redundant to the above paragraph.

Reply:
I've merely changed the format here, not the content. 

The 'Note' was previously inside the code block and all I've tried to do is use 'Note' consistently across the entire doc.

Reply:
I understand, but many of the `Note` callouts are a bit unnecessary, there are now 6. They can distract more than they help with basic things that would otherwise flow well in a paragraph, and the grayed-out presentation of them (in Github's current dark mode) may actually lessen their content's impact/readability.

While we are updating this doc, I figure redundant bits of information can be removed.

Reply:
I agree - I don't think this is a very helpful 'note'. I will remove.

Reply:
Removed. Please mark as resolved.